api/stream.go

```diff
@@ -593,12 +582,15 @@ func (r streamRoutes) deleteVideoSection(c *gin.Context) {
 		return
 	}
 
-	go func() {
-		err := DeleteVideoSectionImage(r.DaoWrapper.WorkerDao, file.Path)
-		if err != nil {
-			log.WithError(err).Error("failed to generate video section images")
-		}
-	}()
+	file, err := r.FileDao.GetFileById(fmt.Sprintf("%d", old.FileID))
+	if err == nil {
+		go func() {
```

Comment:
Just curious: Why is `DeleteVideoSectionImage` non-blocking BTW? Is it too slow otherwise?

Reply:
`DeleteVideoSectionImage` is a request to a worker, therefore yes.